packages/core/src/internal-components/overlay/overlay.element.ts

```diff
@@ -223,7 +222,7 @@ export class CdsInternalStaticOverlay extends CdsBaseFocusTrap {
     false: AnimationModalEnterName,
   },
 })
-export class CdsInternalOverlay extends CdsInternalStaticOverlay implements Animatable {
```

Comment:
No longer need this `implements`. The decorator will complain if `cdsMotion` or `cdsMotionChange` are not present.